input/resources/ValueSet-name-context.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should have a registered OID as an identifier, see https://github.com/hl7au/au-fhir-base/wiki#terminology-resource-conventions